ledsign/sqsread.js

```diff
@@ -0,0 +1,95 @@
+const express = require('express');
+const axios = require('axios');
+const app = express();
+const http = require("http");
+const hostname = 'X.X.X.X';
+const port = 8083;
+const LED_URL = 'XXXXXXXXXXXXX';
+
+//Create HTTP server and listen on port 3000 for requests
+const server = http.Server(app);
+
+// send as part of request body which isn't present in url
+app.use(express.json());
+
+app.get('/healthCheck', (req, res) => {
+  axios.get(LED_URL + 'api/health-check')
+    .then(response => {
+      res.send(response.data);
+    })
+    .catch(error => {
+      console.error('ERROR', error);
+    });
+});
+
+app.post('/updateSignText', (req, res) => {
+  axios.post(LED_URL + 'api/update-sign', req.body)
+    .then(response => {
+      res.send(response.data);
+    })
+    .catch(error => {
+      console.error('ERROR', error);
+    });
+});
+
+//listen for request on port 8083, and as a callback function have the port listened on logged
+server.listen(port, hostname, () => {
```

Comment:
I think you can remove all the server stuff. this file just listens for messages pushed on SQS and then acts accordingly.